fuel-vm/src/storage/interpreter.rs

```diff
@@ -17,8 +17,8 @@ pub trait InterpreterStorage:
     + StorageSize<ContractsRawCode, Error = Self::DataError>
     + StorageRead<ContractsRawCode, Error = Self::DataError>
     + StorageMutate<ContractsInfo, Error = Self::DataError>
-    + MerkleRootStorage<ContractId, ContractsAssets, Error = Self::DataError>
     + MerkleRootStorage<ContractId, ContractsState, Error = Self::DataError>
+    + ContractsAssetsStorage<Error = Self::DataError>
```

Comment:
Should we also do the same thing for state, and have a new `ContractsStateStorage` trait?

Reply:
Probably but this call doesn't use that so it's maybe good for a follow up?

Reply:
Yeah, let's do that in follow up, because we can apply the same ideas for for "contracts" methods